README.md

```diff
@@ -221,17 +219,28 @@ Commit changes to `config/secrets.yml.enc` to a branch
 
 Raise a PR and merge to `master`
 
-# Data loading via HTTP
+## Ensure `registry-data` respository matches data to be loaded into Beta
+
+Create a pull-request in `registry-data` moving all register data associated with the register to be promoted into the `/beta` directory. This should include any new fields added to the `/field` subdirectory, a new file in the `/register` subdirectory as well as a new subdirectory for the new register itself. Merge this into `master`.
```

Comment:
unfortunately there is no PR that has done this correctly yet since the process changed. I can add this after we have done the process successfully once!